js/game.js

```diff
@@ -7,36 +7,57 @@ import { Player } from './player.js';
 
 export class Game {
   constructor(inPlayers) {
-    this.round = 0;
+    this.round = 1;
     this.players = inPlayers.map((name) => new Player(name));
     this.board = new Board();
     this.data = new DataManager();
   }
 
-  update() {
+  update(clueId, playerGuess) {
+    const isCorrect = this.checkAnswer(clueId, playerGuess);
+    this.data[clueId].available = false;
+    this.updatePlayerScore(isCorrect, clueId);
+    this.rotateCurrentPlayer();
 
+    this.updateRound();
+    this.determineWinner();
   }
 
-  checkAnswer() {
+  checkAnswer(clueId, playerGuess) {
+    const correctAnswer = this.data[clueId].answer;
+    
+    return playerGuess === correctAnswer;
 
   }
   
-  updatePlayerScore(clueId) {
-    this.players[0].score += this.data[clueId].value;
+  updatePlayerScore(shouldIncrement, clueId) {
+    if (shouldIncrement) {
+      this.players[0].score += this.data[clueId].value;
+    } else {
+      this.players[0].score -= this.data[clueId].value;
+    }
 
   }
 
   rotateCurrentPlayer() {
-
+    this.players.push(this.players.shift());
   }
 
   determineWinner() {
 
   }
 
   updateRound() {
-
+    let bool = false;
+    
+    for (let i = 0; i < 20; i++) {
+      if (!bool) {
+        bool = this.data[i].available;
+      }
+    }
+
+    if (!bool) {
+      this.round++;
+    }
```

Comment:
Having a hard time following this method...might just be because `bool` is a kind of vague variable name? Not exactly sure what we're checking for in that for loop. What does `this.data[i].available` represent?